website_sale_vat_required/website_order_view.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
 <data>
-    <template id="saas_users" inherit_id="website_sale.checkout">
-        <label class="control-label" for="vat" position="attributes">
+    <template id="vat_checkout" inherit_id="website_sale.checkout">
+        <xpath expr="//div[@class='container oe_website_sale']/form/div[@class='row']/div[@class='col-md-8 oe_cart']/div[@class='row']/div[4]/label" position="attributes">
```

Comment:
Can you even add an option="True" in order to have this template as optional with the module installed.

Why? you can simply add a test on your own activating it but you will not brake other tests that inherit the same view buit do not depend of this feature.